src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxr.h

```diff
@@ -30,6 +30,8 @@ typedef int(*hostfxr_get_runtime_property_value_fn)(void* host_context_handle, P
 typedef int(*hostfxr_run_app_fn)(void* host_context_handle);
 typedef int(*hostfxr_close_fn)(void* hostfxr_context_handle);
 
+#define APPARGNOTRUNNABLE 0x80008094
```

Comment:
Use a const instead?

Reply:
@elinor-fung is this error code from hostfxr safe for us to rely on?
https://github.com/dotnet/runtime/blob/fd9b3d101096bf0d8b65fa2cc7ebe6e84d2d280a/src/native/corehost/fxr/command_line.cpp#L161-L181

Reply:
For `hostfxr_get_native_search_directories` (which looks like the case here), yes. We explicitly test that passing in args pointing to an app that doesn't exist for that API should return `AppArgNotRunnable`.

For other APIs (for example `hostfxr_main_startupinfo`), we fall back to trying to do things with the SDK and would end up with a different error code.